src/CardArea/CardArea.js

```diff
@@ -61,12 +61,21 @@ class CardArea extends Component {
   }
   
   expandDeck = (deckObj) => {
-    const selectedDeck = this.props.getExpandedDeckInfo(deckObj)
-    this.setState({ selectedDeck });
+    const cardsInDeck = this.props.getExpandedDeckInfo(deckObj);
```

Comment:
I renamed this variable to `cardsInDeck` to make it more clear to myself that it was an array of cards and not a deck object.